src/main/java/org/cryptomator/ui/error/ErrorController.java

```diff
@@ -146,7 +147,8 @@ public void lookUpSolution() {
 		askedForLookupDatabasePermission.set(true);
 		HttpClient httpClient = HttpClient.newBuilder().version(HttpClient.Version.HTTP_1_1).build();
 		HttpRequest httpRequest = HttpRequest.newBuilder()//
-				.uri(URI.create(ERROR_CODES_URL))//
+				.header("User-Agent", USER_AGENT_VERSION_FORMAT.formatted(environment.getAppVersion(),environment.getBuildNumber().orElse("undefined")))
+				.uri(URI.create(ERROR_CODES_URL_FORMAT.formatted(URLEncoder.encode(errorCode.toString(),StandardCharsets.UTF_8))))//
```

Comment:
Just noticed that we are using `:` in the error code. These need to be replaced, use an [URLEncoder](https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/net/URLEncoder.html) for this task.

Reply:
whoopsie, it is already present. 🙈